eclair-core/src/main/scala/fr/acinq/eclair/io/Server.scala

```diff
@@ -52,6 +53,7 @@ class Server(nodeParams: NodeParams, authenticator: ActorRef, address: InetSocke
   def listening(listener: ActorRef): Receive = {
     case Connected(remote, _) =>
       log.info(s"connected to $remote")
+      Kamon.counter("peers.connecting.count").withTag("state", "connected").increment()
```

Comment:
This is the same counter used in `Authenticator`. So definitely worth centralizing somewhere to make sure these two actors keep using the same counter correctly